pod2daemon/csidriver/driver/node.go

```diff
@@ -107,7 +109,19 @@ func (ns *nodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnp
 	podInfo, err := ns.retrievePodInfoFromFile(req.VolumeId)
 	if err != nil {
 		log.WithError(err).Error("Unable to retrieve pod info")
-		return nil, status.Errorf(codes.Internal, "Unable to retrieve pod info: %s", err)
+		// If the pod-info file is missing, it's likely the container volumes were already unmounted.
+		// This can be the case when a node-restart occurs as a pod is terminating: upon reboot, the
+		// pod container dir will not be mounted, but we still receive a CSI call to unmount volumes.
+		if !errors.Is(err, fs.ErrNotExist) {
+			return nil, status.Errorf(codes.Internal, "Unable to retrieve pod info: %s", err)
+		}
+		log.Info("Ignoring ErrNotExist and continuing with unmount")
```

Comment:
```suggestion
		log.Info("Pod information file is already gone, continuing with unmount.")
```
Might rephrase to be a little more user accessible.  OK, the user might not know what the pod info file is but the log sortof hints that we were trying to get rid of it.